src/scripts/ic7/malaria/malaria_analysis.py

```diff
@@ -93,7 +93,7 @@ def get_malaria_database(load_data_from_raw_files: bool = True) -> Analysis:
 
 
 def get_malaria_analysis(
-        load_data_from_raw_files: bool = True,
+        load_data_from_raw_files: bool = False,
```

Comment:
these changes are in the file for the IC7 analysis. I suspect we should roll these back?